tests/ProjectConfigTest.php

```diff
@@ -328,6 +330,23 @@ public function testInit()
         $this->assertEquals($expectedVariation, $actualVariation);
     }
 
+    public function testExceptionThrownForUnsupportedVersion()
+    {
+        /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
nit. Remove so many backslashes and ===. Just write plain comment as in other methods